package-lock.json

```diff
@@ -12782,37 +12783,29 @@
     "node_modules/fsevents/node_modules/abbrev": {
       "version": "1.1.1",
       "integrity": "sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q==",
-      "dev": true,
       "inBundle": true,
-      "license": "ISC",
-      "optional": true
```

Comment:
Any idea if these changes in the lock file are okay? Not sure why npm changed it. Used `npm i` (npm v7.24.2)